OrtheusC.c

```diff
@@ -332,6 +335,47 @@ struct List *parseSequences(char **argv, int64_t inputSequenceNumber, int64_t *s
     return seqs;
 }
 
+struct List *parseSequencesFromFile(char *inputSequencesFile, int64_t *seqNo, int64_t **seqLengthsA) {
```

Comment:
Add comments here and in the previous function to explain both have to be kept in sync

Reply:
Although not the most efficient solution, I have managed to merge these two methods into one.